### DISCUSSION TITLE
Strange behaviour with the a key

### DISCUSSION THREAD

**Discussion by kenNash** [evan-liu/karabiner.ts#149] [Q&A]
I'm getting crazy.

The following code works perfectly… except for the `a`, which produces absolutely nothing.

```
import {
  layer,
  toKey,
  withModifier,
  writeToProfile,
} from 'karabiner.ts'

writeToProfile(
  'TS',
  [
    layer_caps_lock(),
  ]
)

function layer_caps_lock() {
  return layer('⇪')
    .notification("⇪ Layer")
    .manipulators([
      withModifier('??')({
        j: toKey('←'),
        k: toKey('→'),
        l: toKey('↑'),
        'semicolon': toKey('↓'),
        //
        a: toKey('‹⌥'), // Alt
        s: toKey('‹⇧'), // Shift
        d: toKey('‹⌘'), // commanD
      })
  ])
}
```

If I change `a: toKey('‹⌥'), // Alt` into `b: toKey('‹⌥'), // Alt`, it works.
If I change `a: toKey('‹⌥'), // Alt` into `a: toKey('a'), // a`, it produces the letter `a`, as expected.
If I press `⌥` instead of `a`, it works.
EventViewer says that `⇪a` is `left_option`.

Any idea?

**Comment by evan-liu** [comment]
Can you open the "Karabiner-EventViewer" app and see the output when you press `a` v.s. `d` on the layer?

**Comment by kenNash** [comment]
`a`: `left_option`
`d`: `left_command`

**Reply by evan-liu** [reply]
That means `a` triggered `left_option` as configured. Which part is not working on your side?

**Comment by kenNash** [comment]
While holding down `⇪`, then `a`, the four keys `jklò` (`ò` is seen as `semicolon` by Karabiner-EventViewer on my Italian keyboard) do absolutely nothing.
`s` and `d` behave as expected.

My code works well on your machine (provided you change `semicolon` with the key you have on the right of `L`)?

**Reply by evan-liu** [reply]
I tried on my machine, and it has the same issue, but I cannot figure out why 😅, sorry.

**Comment by kenNash** [comment]
OK, thank you anyway!
